src/main/java/edu/ucsb/cs156/frontiers/controllers/CoursesController.java

```diff
@@ -155,4 +162,75 @@ public Object handleInvalidInstallationType(Throwable e) {
     }
 
 
+
+    /**
+     * This method looks up a current user and gets their email.
+     * With that email, it that email on every course roster,
+     * and when it appears, notes/stores course id
+     * We then return all courses for those course ids, with relevant
+     * fields for the student.
+     * 
+     * Relevant fields are: id, installationId, orgName, courseName, term, school
+     * For each course, return the status that the student is in
+     */
+    
+    @Operation(summary = "Get all courses for a student")
+    @PreAuthorize("hasRole('ROLE_STUDENT')")
```

Comment:
```suggestion
    @PreAuthorize("hasRole('ROLE_USER')")
```

There is no ROLE_STUDENT in this code base.